packages/auto_mappr/lib/src/generator/auto_mappr_generator.dart

```diff
@@ -193,6 +193,7 @@ class AutoMapprGenerator extends GeneratorForAnnotation<annotation.AutoMappr> {
               ),
           ];
         })
+        // ignore: avoid-slow-collection-methods, fix dcm without refactoring
```

Comment:
Comment does not make sense, no? This is IMO ok to use here, since we create multiple type mappings from each .map iteration and then we need to flatten it. Maybe this would be OK and faster than this sync* function?

```dart
return [
  for (item in mappers.map())
    ...item,
];
```